docs/reference/errors-and-warnings/NU3043.md

```diff
@@ -0,0 +1,39 @@
+---
+title: NuGet Warning NU3043
+description: NU3043 warning code
+author: kartheekp-ms
+ms.date: 09/11/2024
+ms.topic: reference
+f1_keywords: 
+  - "NU3043"
+---
+
+# NuGet Warning NU3043
+
+`dotnet nuget sign command` - Invalid value for `--certificate-fingerprint` option. The value must be a SHA-256, SHA-384, or SHA-512 certificate fingerprint (in hexadecimal).
+
+`NuGet.exe sign command` - Invalid value for `CertificateFingerprint` option. The value must be a SHA-256, SHA-384, or SHA-512 certificate fingerprint (in hexadecimal).
+
+> [!NOTE]
+> This warning will be promoted to an error around the .NET 10 timeframe.
+
+## Solution
```

Comment:
All other warning pages has an `## Issue` header, that explains why the warning exists, before the solution section

Reply:
Great catch. Fixed in the latest commit.